web-common/src/Halogen/Analytics.purs

```diff
@@ -1,17 +1,15 @@
 module Halogen.Analytics where
 
-import Prelude
 import Analytics (class IsEvent, analyticsTracking)
 import Effect.Class (class MonadEffect, liftEffect)
 import Halogen (HalogenM)
-import Prim.TypeError (class Warn, Text)
+import Prelude (discard, ($))
 
-handleActionWithAnalyticsTracking ::
+withAnalytics ::
   forall state action slots message m a.
-  Warn (Text "TODO: Rename handleActionWithAnalyticsTracking to withAnalytics") =>
   MonadEffect m =>
   IsEvent action =>
   (action -> HalogenM state action slots message m a) -> action -> HalogenM state action slots message m a
-handleActionWithAnalyticsTracking handleAction' action = do
+withAnalytics handleAction' action = do
```

Comment:
Nitpick, but can we call `handleAction'` just `handler`?

Reply:
Okay.